packages/node-resizin/src/index.ts

```diff
@@ -0,0 +1,6 @@
+import { uploadFactory } from 'resizin';
+import FormData from 'form-data';
+
+(global as any).FormData = FormData;
```

Comment:
Whats this syntax?

Reply:
global namespace (which is node global of course) has a type definition which does not contain the `FormData` class. To be able to put it on it, first we need to extend type of global namespace. But you're right, I'm going to adjust it to make it more obvious.